File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/PackagesUpdatePackageUseByDateRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class PackagesUpdatePackageUseByDateRequestItem {
    @JsonProperty("ExpirationDate")
    public String expirationDate;
    @JsonProperty("Label")
    public String label;
    @JsonProperty("SellByDate")
    public String sellByDate;
    @JsonProperty("UseByDate")
    public String useByDate;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PackagesUpdatePackageUseByDateRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PackagesUpdatePackageUseByDateRequestItem(
    @JsonProperty("expirationDate")
    val expirationDate: String? = null,
    @JsonProperty("label")
    val label: String? = null,
    @JsonProperty("sellByDate")
    val sellByDate: String? = null,
    @JsonProperty("useByDate")
    val useByDate: String? = null
)
